packages/css/src/ordered-list/README.md

```diff
@@ -0,0 +1,15 @@
+# Ordered List
+
+## Introductie
+
+Een lijst is een verticale groep gerelateerde inhoud.
```

Comment:
This is a little generic, maybe specify that it's hierarchical?

Reply:
Formatted it more like an introduction paragraph like https://www.amsterdam.nl/digitalservices/design-system/atomen-moleculen/list/